pkg/githubclient/client_repository.go

```diff
@@ -52,3 +53,175 @@ func (c *Client) ListRepositories(ctx context.Context, owner string) ([]Reposito
 
 	return repos, nil
 }
+
+func (c *Client) GetRepository(ctx context.Context, owner, repo string) (*github.Repository, error) {
+	c.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("get repository %s/%s", owner, repo))
```

Comment:
I'd strongly prefer a user friendly output to stdout instead of log messages.

Reply:
Happy to replace this with a friendly output, but do you have any suggestion on what to use?